### ISSUE TITLE
Can't find cropformer.yml

### ISSUE THREAD

**Issue by runzewei99** [jiekesen/Cropformer#2]
Teacher or Senior, I'm using your software for learning at the moment, but I can't find the corresponding cropformer.yml file.

**Comment by noobylf** [comment]
yeah, I meet the same question, and do you kown does the encoding method of the gene file have to be from 0 to 9 to use this model to run the data?

**Comment by jiekesen** [comment]
Thanks for your question, we have uploaded cropformer.yml
